acrobat/blocks/dc-converter-widget/dc-converter-widget.js

```diff
@@ -20,21 +20,24 @@ const verbToRedirectLinkSuffix =  {
 };
 export default function init(element) {
   const widget = element;
-  let WIDGET_ENV = 'https://dev.acrobat.adobe.com/dc-hosted/2.37.2_1.165.0/dc-app-launcher.js';
+  let DC_WIDGET_VERSION_FALLBACK = '2.37.2_1.165.0';
+  let DC_GENERATE_CACHE_VERSION_FALLBACK = '1.165.0';
+  let DC_WIDGET_VERSION = document.querySelector('meta[name="dc-widget-version"]')?.getAttribute('content') || DC_WIDGET_VERSION_FALLBACK;
+  let DC_GENERATE_CACHE_VERSION = document.querySelector('meta[name="dc-generate-cache-version"]')?.getAttribute('content') || DC_GENERATE_CACHE_VERSION_FALLBACK;
+  let WIDGET_ENV = `https://dev.acrobat.adobe.com/dc-hosted/${DC_WIDGET_VERSION}/dc-app-launcher.js`;
   let ENV = 'dev';
   let REDIRECT_URL = '';
   let DC_GENERATE_CACHE_URL = '';
-
   if (window.location.hostname === 'main--dc--adobecom.hlx.page'
     || window.location.hostname === 'main--dc--adobecom.hlx.live'
     || window.location.hostname === 'www.adobe.com') {
-    WIDGET_ENV = 'https://documentcloud.adobe.com/dc-hosted/2.37.2_1.165.0/dc-app-launcher.js';
+    WIDGET_ENV = `https://documentcloud.adobe.com/dc-hosted/${DC_WIDGET_VERSION}/dc-app-launcher.js`;
```

Comment:
this url is also available in the metadata, along with dev and prod variants

Reply:
In which metadata file is that stored? i don't see it here https://adobe.sharepoint.com/:x:/r/sites/adobecom/_layouts/15/Doc.aspx?sourcedoc=%7BBC0B7771-604F-4E0F-8319-FA0D6AEEFCBD%7D&file=metadata.xlsx&action=default&mobileredirect=true

Reply:
This metadata, which I guess would need to be carried over to sharepoint somehow: https://acrobat.adobe.com/dc-hosted/private/info.json